src/solvers/CMakeLists.txt

```diff
@@ -6,15 +6,15 @@
 
 set(solvers_sources
     base_solver.cpp
-    quasistatic_solver.cpp
+    nonlinear_solid_solver.cpp
```

Comment:
if I want to do dynamics with a nonlinear material, am I using nonlinear_solid_solver or dynamic_solver?

Reply:
This should replace both `dynamic_solver` and `quasistatic_solver`.